esp_encrypted_img/idf_component.yml

```diff
@@ -4,3 +4,5 @@ url: https://github.com/espressif/idf-extra-components/tree/master/esp_encrypted
 dependencies:
   idf:
     version: ">=4.4"
+  espressif/esp_secure_cert_mgr:
+    version: '*'
```

Comment:
Lets set rule to allow only minor or patch version. Major version may break the API.

Reply:
Added in version check